src/raqm.c

```diff
@@ -279,6 +262,61 @@ _raqm_compare_text_info (_raqm_text_info a,
   return true;
 }
 
+static void
+_raqm_free_text(raqm_t* rq)
+{
+    free(rq->text);
```

Comment:
```suggestion
    free (rq->text);
```

Reply:
Indentation is also off in this function.

Reply:
Fixed